packages/trace-viewer/src/ui/metadataView.tsx

```diff
@@ -25,6 +25,12 @@ export const MetadataView: React.FunctionComponent<{
   if (!model)
     return <></>;
   return <div className='metadata-view vbox'>
+    {model.baseURL && (
+      <>
+        <div className='call-section' style={{ paddingTop: 2 }}>Time</div>
+        <div className='call-line'>baseURL:<a className='call-value string' href={model.baseURL} title={model.baseURL} target="_blank">{model.baseURL}</a></div>
```

Comment:
```suggestion
        <div className='call-line'>baseURL:<a className='call-value string' href={model.baseURL} title={model.baseURL} target='_blank' rel='noopener noreferrer'>{model.baseURL}</a></div>
```

I wonder why lint does not argue against double quotes...

Reply:
Because there's a specail `jsx-quote` rule that we don't have enabled yet. Went ahead and fixed that in https://github.com/microsoft/playwright/pull/31855, so we don't risk breaking the space-time continuum.